chpl/chpl-service/src/test/java/gov/healthit/chpl/manager/impl/CertifiedProductDetailsManagerTest.java

```diff
@@ -34,6 +34,7 @@
 import gov.healthit.chpl.domain.InheritedCertificationStatus;
 import gov.healthit.chpl.domain.MacraMeasure;
 import gov.healthit.chpl.manager.CertifiedProductDetailsManager;
+import gov.healthit.chpl.manager.CertifiedProductManager;
```

Comment:
This is the CPDManager test, not the CPManager test. These tests should be in a different file